src/vs/platform/terminal/common/terminal.ts

```diff
@@ -62,6 +62,7 @@ export const enum TerminalSettingId {
 	LetterSpacing = 'terminal.integrated.letterSpacing',
 	LineHeight = 'terminal.integrated.lineHeight',
 	MinimumContrastRatio = 'terminal.integrated.minimumContrastRatio',
+	TabSize = 'terminal.integrated.tabSize',
```

Comment:
Let's align with xterm.js and use `tabStopWidth` as it's the terminology used on https://invisible-island.net/xterm/ctlseqs/ctlseqs.html and iTerm2 and other terminals.